src/state/application/hooks.ts

```diff
@@ -126,6 +126,37 @@ export function useAddPopup(): (content: PopupContent, key?: string) => void {
   )
 }
 
+export enum NotificationType {
+  SUCCESS,
+  ERROR,
+  WARNING,
+}
+export const useNotify = () => {
+  const dispatch = useDispatch()
+  return useCallback(
+    ({
+      title,
+      type = NotificationType.ERROR,
```

Comment:
why only error type for notify? name is confusing me

Reply:
this is default value: can also have warning, success

Reply:
But useNotify doesnt have any props?

Reply:
it just like useAddPopup. but this update to 
+simplify params 
<img width="898" alt="image" src="https://user-images.githubusercontent.com/33005392/182765214-fbd01e32-4278-4b35-9732-81b781bd0127.png">
+enable with many `type: success | warning | error` instead of `success: true | false`